src/components/Button/index.tsx

```diff
@@ -118,6 +118,9 @@ type ButtonProps = Partial<ChildrenProps> & {
 
     /** Whether the button should use split style or not */
     isSplitButton?: boolean;
+
+    /** Whether button's content should be centered */
+    centeredContent?: boolean;
```

Comment:
I know its a minor thing and NAB
but a bool prop to me should be named using a pattern like: `isContentCentered`/`shouldCenterContent`.

If I just see the name `centeredContent` then I would presume there is some content _inside_ this property (like a piece of JSX or icon or something...

Reply:
right, renamed